exercises/grade-school/src/main/kotlin/GradeSchool.kt

```diff
@@ -0,0 +1,17 @@
+class School {
```

Comment:
Why not naming the kt-file after the class? Isn't it a recommended Kotlin practice?
Or each solution file should be named after the slug?

Reply:
Technically, `School` represents some entity here.

But I left all entity-names to avoid fixing tests as well.